packages/framework/tests/Unit/Component/Cron/CronFacadeTest.php

```diff
@@ -132,18 +129,6 @@ private function mockCronModuleFacade()
         return $this->createMock(CronModuleFacade::class);
     }
 
-    /**
-     * @param \PHPUnit\Framework\MockObject\MockObject $mock
-     * @param string $methodName
-     * @param string $serviceId
-     */
-    private function expectMethodCallWithCronModuleConfig($mock, $methodName, $serviceId)
-    {
-        $mock->expects($this->atLeastOnce())
```

Comment:
With this you removed the test-case, or? Is it right?

Reply:
Ou, yes, I expect discussion. In this issue (https://github.com/sebastianbergmann/phpunit/issues/3339#issuecomment-428843322) is wrote, that we should not tested implementation details, so I removed this test :smile: